### PR TITLE
Add code linting to CI workflow

### PR DESCRIPTION
**Pull request by sergeyklay**
- Add code linting to CI workflow
- Fix code style in `mac_helpers`

/cc @tasn

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,2 +1,7 @@
 [flake8]
 max-line-length = 120
+
+# A list of mappings of files and the codes that should be ignored for
+# the entirety of the file:
+per-file-ignores =
```
Comment (tasn) [inline .flake8:6]: I'd much rather have it in the file itself than the .flake8 config.

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   lint:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```
Comment (tasn) [inline .github/workflows/lint.yml:14]: The reason why we are using ubuntu-18.04 for everything, is that we need it in order to generate binaries that work across Linux versions. So we use it everywhere for consistency.
```suggestion
    runs-on: ubuntu-18.04
```

File: .github/workflows/manual.yml [hunk 3]
```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-18.04, windows-latest, macos-latest]
+        os: [ubuntu-latest, windows-latest, macos-latest]
```
Comment (tasn) [inline .github/workflows/manual.yml:14]: It should be 18.04.
```suggestion
        os: [ubuntu-18.04, windows-latest, macos-latest]
```

**Review by tasn — CHANGES_REQUESTED** [review]

**Comment by sergeyklay** [comment]
@tasn Could you please take a look :)

**Review by tasn — COMMENTED** [review]
Sorry for being a bit annoying, but this change is 40% doing what it says, and the rest is adding changelog information and other unrelated stuff. Could you please split to two PRs?

**Comment by sergeyklay** [comment]
Yes, of course. I'll do today a bit later

**Comment by tasn** [comment]
Thanks! Sorry about that, just trying to keep things organised, both for myself and other contributors! \o/

**Comment by tasn** [comment]
Much better, thanks!

**Comment by sergeyklay** [comment]
You're welcome :)

**Pull request merged**